### DISCUSSION TITLE
Various

### DISCUSSION THREAD

**Discussion by heXc-me** [jilleb/MQB-FPA#5] [Exploration and research]
1) Looks like i´m the first and only one idiot here, some comparisons.

START_ADDRESS="0x000A80(2688)  = VCP Wear recognition (Müdigkeitserkennung)
START_ADDRESS="0x0000B80 (2944)  = ODIS FPA Datensatz
                             0x0001380 (4992)  = VCP Datensatz -> Start/stop
START_ADDRESS="0x0002880 (10368) = ODIS Start/Stop
                             0x0002388 (9096)  = VCP Drive Profil xyz,Start-Stop settings
START_ADDRESS="0x0001C80 (7296  = ODIS Energie Managment / Generator
                             0x0001B80 (7040) = VCP Etron Reichweite, GTE Reichweite

2)  for Starting Identer List at 0x00-0x04

OJ00d -> MK 7.5 FL GTI (LHD,no DCC,ACC,2WD) + MK7-5 FL R(RHD,DCC,ACC,4WD)

3) Cupra FPA mode is complete missing,Slider Menu also. Need help ?

**Comment by jilleb** [comment]
1: you need the data from START_ADDRESS="0x0000B80
2: I'll make a sheet with dataset versions and  which are used on what car. Thanks!
3: dcc slider only exists on individual mode, it's marked in the template. Cupra is in the template as well (is the same as Race)

**Reply by batal131** [reply]
Not the best place to answer, but...

I have few datasets (both from Kodiaq). How can I share them with you?

**Reply by langers2k** [reply]
You should be able to attach files by dragging them into the reply box.

You might need to zip them up first as I think the file types are restricted.
